Sources/Mapper.swift

```diff
@@ -332,7 +339,7 @@ public struct Mapper {
      */
     @warn_unused_result
     public func from<T>(field: String, transformation: AnyObject? throws -> T) rethrows -> T {
-        return try transformation(self.JSONFromField(field))
+        return try transformation(try? self.JSONFromField(field))
```

Comment:
I'd rather not swallow this error, but there are 2 problems here. First if we didn't, we'd be making a breaking API change from `AnyObject?` to `AnyObject`. Overall I'm ok with doing that at some point, but not just yet. Second we couldn't utilize `rethrows` here, which is also too bad. But the first one is the bigger problem for now.